### DISCUSSION TITLE
Next updates (Poll)

### DISCUSSION THREAD

**Discussion by simmmpleweb** [horizon-ui/horizon-ui-chakra#4] [Polls]
Hello, guys! We are working now on the next updates and we thought that it would be easier for both us & you to know your needs and wishes for all that means Horizon UI, as the next updates will consist of what the community wants the most! ⚡️

**Comment by rzmz** [comment]
TypeScript.

**Reply by simmmpleweb** [reply]
Hello, @rzmz!  The most awaited day and the most wanted & requested version of Horizon, is now officially here! Horizon UI TypeScript can be downloaded & bought now from our website! It's one of our first products made with TypeScript, so if you guys have any ideas of how we can improve it code-wise, it will be more than great! Until that, enjoy and expand your horizons with the new Horizon TypeScript! ❤️

Free version: https://horizon-ui.com/#version
PRO Version: https://horizon-ui.com/pro/#pricing
